devicetypes/sky-nie/min-smart-plug.src/min-smart-plug.groovy

```diff
@@ -383,4 +418,14 @@ private logDebug(msg) {
 
 private logTrace(msg) {
     log.trace "$msg"
-}
\ No newline at end of file
+}
+
+private isButtonAvailable() {
```

Comment:
@sky-nie Since You don't support a child device anymore, is this function still needed (and all related device settings) ?

Reply:
I keep the function "isButtonAvailable" because the preferences need it to distinguish the difference number and the ledOptions settings